docs/analyse_database/README.md

```diff
@@ -0,0 +1,63 @@
+# Analyse database
+The analyse database (gob_analyse) is used by basisinformatie to perform analyses on the GOB data.
+Employees of basisinformatie have direct access to this database. Their access is controlled with
+Postgres roles. This directory contains the necessary SQL scripts. This file explains how to use them.
+
+## Design
+### Schemas and default roles
+All registrations in GOB have their own schemas in the analyse database. There is a bag schema, brk schema,
+gebieden, etc. For each schema there is a corresponding _read role (bag_read, brk_read, gebieden_read,
+etc). This read role has, as the name suggests, read permissions on the schema. There are no write roles
+for these schemas.
+
+There is one extra schema called 'maatwerkbasisinformatie', where (some) basisinformatie employees have
+write permissions. This is their playground where they can place the results of their analyses. Two roles
+exist for this schema: maatwerkbasisinformatie_read and maatwerkbasisinformatie_write.
+
+The next section explains how these roles are used in practice.
+
+### Access control
+Two main roles are used to control access to new analyse database users: agm_basis and agm_plus. Users
+are granted either only the agm_basis role or both the agm_basis AND agm_plus roles.
+
+- The agm_basis role provides read access to all registrations except BRK and read access to the
+maatwerkbasisinformatie schema.
+- The agm_plus role provides read access to BRK and write access to the maatwerkbasisinformatie schema.
+
+## SQL scripts in this directory
+### default_roles.sql
+This file contains the SQL to create the default roles, including agm_basis and agm_plus. The SQL in this
+file should only have to be run when creating a new analyse database from scratch. Only when a new
```

Comment:
this script should be executed by the user that creates the tables, gob_analyse in our case. If another user executes this script the read permissions will not be set automatically on table creation

Reply:
Good one, added :)